include/awkward/python/content.h

```diff
@@ -76,39 +76,16 @@ namespace {
   class EmptyBuffersContainer: public ak::BuffersContainer {
   public:
     void*
-      empty_buffer(const std::string& name, int64_t num_bytes) override {
+      empty_buffer(const std::string&, int64_t) override {
         return nullptr;
       }
 
     void
-      copy_buffer(const std::string& name, const void* source, int64_t num_bytes) override { }
+      copy_buffer(const std::string&, const void*, int64_t) override { }
 
     void
-      full_buffer(const std::string& name, int64_t length, int64_t value, const std::string& dtype) override { }
+      full_buffer(const std::string&, int64_t, int64_t, const std::string&) override { }
   };
-
-  /// @brief Turns the accumulated data into a Content array.
-  ///
-  /// This operation only converts Builder nodes into Content nodes; the
-  /// buffers holding array data are shared between the Builder and the
-  /// Content. Hence, taking a snapshot is a constant-time operation.
-  ///
-  /// It is safe to take multiple snapshots while accumulating data. The
-  /// shared buffers are only appended to, which affects elements beyond
-  /// the limited view of old snapshots.
-  py::object
-  builder_snapshot(const ak::BuilderPtr builder) {
```

Comment:
This function is no longer used.

Reply:
Indeed, it was needed for an `ArrayBuilder` and a legacy `LayoutBuilder`.  The first has been ported to a new `LayoutBuilder`, the latter has been removed from the code base.